tests/scripts/ci.py

```diff
@@ -595,11 +601,25 @@ def add_subparser(
             "frontend": ("run frontend tests", ["./tests/scripts/task_python_frontend_cpu.sh"]),
         },
     ),
+    generate_command(
+        name="minimal",
+        help="Run minimal CPU build and test(s)",
+        options={
+            "cpp": CPP_UNITTEST,
+            "unittest": (
+                "run unit tests",
+                [
+                    "./tests/scripts/task_python_unittest.sh",
+                ],
+            ),
+        },
+    ),
     generate_command(
         name="i386",
         help="Run i386 build and test(s)",
         options={
             "cpp": CPP_UNITTEST,
+            "micro_cpp": CPP_MICRO_UNITTEST,
```

Comment:
iirc these turn into command line flags

```suggestion
            "micro-cpp": CPP_MICRO_UNITTEST,
```

Reply:
The `_` are replaced with `-` by the `cli_name` function, so the command line flag available and shown in `--help` is `micro-cpp`.

Strangely, if I replace `micro_cpp` with `micro-cpp` then calling, for example, `python tests/scripts/ci.py cpu --micro-cpp` will only run the build. This might be a limitation in how we generate the argument parsers.